.changeset/fresh-years-float.md

```diff
@@ -0,0 +1,5 @@
+---
+'@farfetched/zod': patch
```

Comment:
patch will make release version 0.0.2, I assume it's better to release it as 0.1.0

```suggestion
'@farfetched/zod': minor
```